src/components/algolia-search/index.tsx

```diff
@@ -63,6 +64,7 @@ export default function AlgoliaSearch<THit extends Hit<unknown>>({
         setAutocompleteState(state)
       },
       navigator,
+      debug: true,
```

Comment:
TODO: remove before merge, keeps the panel open so it can be inspected during development